ecommerce-ios/Sources/Home/View/ProductCollectionView/ProductCollectionView.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  ProductCollectionView.swift
+//  ecommerce-ios
+//
+//  Created by Su T. Nguyen on 20/05/2021.
+//
+
+import SwiftUI
+
+struct ProductCollectionView: View {
+
+    private let viewModel = ProductCollectionViewViewModel()
+
+    private let numberOfColumns: Int = 2
+    private let spacing: CGFloat = 17.0
+
+    private var columns: [GridItem] {
+        let numberOfColumns = CGFloat(self.numberOfColumns)
+        let minimunWidth = ((screenWidth - spacing * (numberOfColumns + 1)) / numberOfColumns).rounded(.down)
+        return [.init(.adaptive(minimum: minimunWidth))]
+    }
```

Comment:
Please take a look at https://github.com/nimblehq/nimble-ecommerce-ios/pull/53#discussion_r635741188 and tell us what you think?